pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -577,7 +577,7 @@ def run(self):
         progress = OSTree.AsyncProgress.new()
         progress.connect('changed', self._pull_progress_cb)
 
-        pull_opts = {'refs': Variant('as', [ref])}
+        pull_opts = {'refs': Variant('as', [ref]), 'flags': Variant('i', OSTree.RepoPullFlags.UNTRUSTED)}
```

Comment:
I think we should include the full rationale from the pull request as a comment here.  Something like:

```suggestion
        # We use the UNTRUSTED flag as a way to help verification for potentially
        # corrupted ISOs - this way ostree will validate the checksum of objects as
        # it writes data.
        pull_opts = {'refs': Variant('as', [ref]), 'flags': Variant('i', OSTree.RepoPullFlags.UNTRUSTED)}
```

Reply:
Added your comment to my commit